practices/handover.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: handover
+stages:
+  - development
+short_description: 
+tags:
+  - efficiency
+  - sustainability
+  - knowledge sharing
+  - knowledge transfer
+  - documentation
+keywords:
+  - agile-handover
+  - knowledge transfer
+  - team building
+  - communication
+  - support 
+covered_by_dx_scanner: false
+---
+
+# Handover
+
+**TL;DR**
+
+Transferring projects from one team to another is often seen as a mammoth task.  It requires a complete transfer of roles and responsibilities. However, following set protocols, having strong communication and reciprocated motivation makes this whole process a lot smoother.
+
+## What Is a Handover?
+
+A handover is a process of transferring the roles and responsibilities of a project from one team/individual to another. Typically, this transfer takes place over the duration of a couple of months, in order to ensure that the receiving team/individual has fully grasped the working of the project, and their roles corresponding to the same. 
+
+## What Is Handover Documentation?
+
+Handover documentation refers to the resources, notes and documents compiled or passed-on by staff members/teams who are about to leave thier current position (either temporarily or permanently). They are 'handed over' to the team/individual that will be succeeding them, in order to help them carry out their newly acquired duties with ease.
+
+## Handover Procedure
+
+A typical handover spans across a couple of months, where the team/individual from the current team is in close communication with the individual/team onboarding onto the project. Having a knowledge transfer stratergy in place is one of the most effective ways of carrying out this transition.
+
+In a broad sense, the handover procedure involves the following steps:
+1. Identify the team/individual that will be on-boarding onto the project
+2. Sync up schedules and timings with the said team/individual in order to find slots that work for both teams
+3. Set-up a recurring meeting in the identified slots
+4. Have a clear stratergy on the key aspects of the projects that you aim to cover
+5. Always leave room for questions, follow-ups and doubts!
+
```

Comment:
DXKB articles usually follow this pattern: 

What Is a %{name} -> Short explanation 
Why You Might Want the %{name} -> Explanation that shows the reader why he wants to do the handover correctly.
Problems the %{name} Helps to Solve -> What problem does Handover help to solve? (I know that this will be harder since Handover solves mainly handover itself :D)
How to Implement the %{name}? -> this seems like the Handover Strategy and Best Practises 
Common Pitfalls of the %{name}
Resources for the %{name}